src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/VtableIndexStubGenerator.cs

```diff
@@ -36,8 +38,15 @@ public static class StepNames
         {
             public const string CalculateStubInformation = nameof(CalculateStubInformation);
             public const string GenerateManagedToNativeStub = nameof(GenerateManagedToNativeStub);
+            public const string GenerateNativeToManagedStub = nameof(GenerateNativeToManagedStub);
         }
 
+        private static readonly ContainingSyntax NativeTypeContainingSyntax = new(
+                                    TokenList(Token(SyntaxKind.InternalKeyword), Token(SyntaxKind.PartialKeyword)),
+                                    SyntaxKind.InterfaceDeclaration,
+                                    Identifier("Native"),
+                                    null);
+
         public void Initialize(IncrementalGeneratorInitializationContext context)
```

Comment:
This function is incredibly dense and I've little to no idea on what it is trying to do. Can we get it commented?

Reply:
Upon walking through this I get it, but I've no confidence I could write it. Up to you if it needs any more clarity.